README.md

```diff
@@ -1,9 +1,10 @@
-# Name Request Online Frontend
+# Name Request UI
```

Comment:
This is what we always call it (and sometimes "Namerequest UI").

We should not refer to "NRO" in the name.

Reply:
Not that I'm aware.